### PR TITLE
ES Page Updates

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: app/views/es-pages/home.js [hunk 1]
```diff
@@ -53,34 +53,15 @@ return ['$location','$scope','fb','$document','ngMeta','$q','$http', function ($
...
-								}
-
-								$scope.carousel=data.response.docs;
-						}).catch(function(error) {
-								console.log('ERROR: ' + error);
-						});
-
-						return pagePromise;
-				}// query
+					var url = 'https://spreadsheets.google.com/feeds/cells/1_RSS-SjMifBEfUetPfr6GX8eJ3M4GMYrTrg4pXFIKkg/4/public/values?alt=json'
```
Comment (blaisefonseca) [inline app/views/es-pages/home.js]: who owns it?

File: app/services/google-sheet-service.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    })
+}
+
+function cleanGoogleSheet(entry, schema, numProps) {
+    var dataRows = []
+    var id = 1
+    var propNamesRow = entry.splice(0, numProps)
+    var propNames = []
+
+    for (let j = 0; j < numProps; j++)
```
Comment (blaisefonseca) [inline app/services/google-sheet-service.js]: let will not work on client

File: app/views/es-pages/home.js (hunk 1 above)
Comment (randyhoulahan) [reply app/views/es-pages/home.js]: IT and Mateusz

File: app/views/es-pages/media.js [hunk 3]
```diff
@@ -26,7 +27,23 @@ return ['$location','$scope','fb','$document','ngMeta', function ($location,$sco
 				ngMeta.setTag('twitter:creator','@CristianaPascaP');
 				ngMeta.setTag('twitter:title',$scope.$root.page.title);
 				ngMeta.setTag('twitter:description',$scope.$root.page.description);
-				ngMeta.setTag('twitter:image','/app/images/es-pages/es3.jpg');
+				ngMeta.setTag('twitter:image','https://attachments.cbd.int/es3.jpg');
 			});
+
+			function getMedia() {
+				var url = 'https://spreadsheets.google.com/feeds/cells/1_RSS-SjMifBEfUetPfr6GX8eJ3M4GMYrTrg4pXFIKkg/5/public/values?alt=json'
```
Comment (sbilodeau) [inline app/views/es-pages/media.js]: Would have been simpler to use the articles?

**Pull request merged**